docs/howto-run-microservices-locally.md

```diff
@@ -29,32 +29,32 @@ The easiest way to start all services is to use the `docker-compose.yml` file lo
    docker-compose up
    ```
 
-   And follow the instructions below to [ensure the services are running](#Ensure-the-services-are-running).
+   And follow the instructions below to [ensure the Microservices are running](#Ensure-the-Microservices-are-running).
 
-## Build and run the services with Visual Studio or VS Code
+## Build and run the Industrial IoT Microservices with Visual Studio or VS Code
 
-1. First, make sure your development tool chain is setup to build the services.  If you have not, [install .NET Core 2.1+][dotnet-install] and any recent edition of Visual Studio (Windows/MacOS) or Visual Studio Code (Windows/MacOS/Linux).
+1. First, make sure your development tool chain is setup to build the Microservices. If you have not, [install .NET Core 2.1+][dotnet-install] and any recent edition of Visual Studio (Windows/MacOS) or Visual Studio Code (Windows/MacOS/Linux).
    - If you already have Visual Studio installed, then ensure you have [.NET Core Tools for Visual Studio 2017][dotnetcore-tools-url] installed (Windows only).
    - If you already have VS Code installed, then ensure you have the [C# for Visual Studio Code (powered by OmniSharp)][omnisharp-url] extension installed. 
 2. Ensure that the `.env` file previously generated by the deployment script is located in the repository root. 
 3. Open the `azure-iiot-components.sln` solution file in Visual Studio or VS Code
 4. Right click on the solution in the solution viewer and select `Properties`. Set the start up projects to be all projects matching `Microsoft.Azure.IIoT.Services.OpcUa.*`.  Set them to "Start".
 5. Start debugging by pressing the "Start" button or hitting F5.
-6. [Ensure the services are running](#Ensure-the-services-are-running)
+6. [Ensure the Microservices are running](#Ensure-the-Microservices-are-running)
```

Comment:
These links will be broken.  They refer to the updated header below.   I would just use Microservices in the header.